x-pack/filebeat/module/panw/panos/_meta/fields.yml

```diff
@@ -291,3 +291,43 @@
       type: keyword
       description: >
         The type of tunnel (either SSLVPN or IPSec).
+
+    - name: connect_method
```

Comment:
Can you add a `default_field: false` key to each of the new fields?

We forgot this for the previous filesets. This is done to avoid hitting the limit of ~10k default fields in the index template for filebeat indices.

Reply:
I updated just the fields I added in this and previous PR.  Do you want me to do more??